### DISCUSSION TITLE
Contributor not showing up in repository

### DISCUSSION THREAD

**Discussion by pkuehnel** [community/community#43288] [Repositories]
### Select Topic Area

Bug

### Body

I have an OpenSource project [TeslaSolarCharger](https://github.com/pkuehnel/TeslaSolarCharger) where @SebaNar added commits in [this pull request](https://github.com/pkuehnel/TeslaSolarCharger/pull/473), but he is not showing up in the main page of my project:

![image](https://user-images.githubusercontent.com/35361981/210519118-214b5e88-55a9-46f9-9692-69dceb1311fa.png)

**Comment by jfullstackdev** [comment]
if you can look at the merge commit history on GitHub, and whether his commits are verified, that could be the case it will not be counted when it's not verified

**Reply by pkuehnel** [reply]
When are commits verified and when not?
